cloud/scope/clients.go

```diff
@@ -24,8 +24,15 @@ func newOscClient() (*OscClient, error) {
 	if secretKey == "" {
 		return nil, errors.New("environment variable OSC_SECRET_KEY is required")
 	}
+	version := os.Getenv("VERSION")
+	if version == "" {
+		return nil, errors.New("Version is required")
+	}
+	config := osc.NewConfiguration()
+	config.Debug = true
+	config.UserAgent = fmt.Sprintf("cluster-api-provider-outscale/%s", version)
 	oscClient := &OscClient{
-		api: osc.NewAPIClient(osc.NewConfiguration()),
+		api: osc.NewAPIClient(config),
 		auth: context.WithValue(context.Background(), osc.ContextAWSv4, osc.AWSv4{
```

Comment:
we have a helper to get creds from env variables or from json file: 
NewConfigEnv() or NewConfigFile , and then methods for configuration() and context() to get creds properly.